dist/api_data.py

```diff
@@ -478,6 +478,10 @@ def __cmp__(self, other):
                 vary depending on the current eviction load''',
                 min=1, max=20),
             ]),
+    Config('readonly', 'false', r'''
+        open connection in read-only mode.  The database must exist.  All
+        methods that may modify a database are disabled.''',
```

Comment:
It feels to me as though read only deserves a page in the reference guide, with an @ref from here. I'm not inclined to document all the API restrictions in detail, but a general overview would be good. Some examples of things I think would be worth mentioning:
* What happens if logging was turned on
* What happens if recovery was required
* What happens if there are LSM trees present